backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveOrCopyDiskCommand.java

```diff
@@ -721,6 +721,14 @@ protected MultipleStorageDomainsValidator createMultipleStorageDomainsValidator(
         return new MultipleStorageDomainsValidator(getStoragePoolId(), sdsToValidate);
     }
 
+    protected StorageDomain getSourceStorageDomain() {
```

Comment:
@ahadas @bennyz 
I believe that the `getSourceStorageDomain()` method can be used in [1] as well (instead of lines 691-696 a simple `getSourceStorageDomain().getStorageName()` will do the same.
And in [2] a `getDestStorageDomain().getStorageName()` is more consistent and readable.

[1] https://github.com/oVirt/ovirt-engine/blob/ovirt-engine-4.5.2/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveOrCopyDiskCommand.java#L691
[2] https://github.com/oVirt/ovirt-engine/blob/ovirt-engine-4.5.2/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/MoveOrCopyDiskCommand.java#L700

Reply:
but it's better to have the results cached